tests/runners/bulk_update_runner.py

```diff
@@ -100,12 +101,16 @@ def __upload_spreadsheet_and_create_submission(self, dataset_fixture, project_uu
         self.submission_envelope = self.ingest_api.envelope(self.submission_id)
 
     def __upload_modified_spreadsheet(self, path_to_spreadsheet):
-        self.ingest_broker.upload(path_to_spreadsheet, is_update=True)
+        submission_uuid = self.submission_envelope.uuid
+        project_uuid = self.project_uuid
+        self.ingest_broker.upload(path_to_spreadsheet, is_update=True, project_uuid=project_uuid,
```

Comment:
Could it be a case when the above `UUIDs` are `None`?

Reply:
Yup, and it should be okay if both are none. But they have values in this case.